pkg/controller/podautoscaler/horizontal_test.go

```diff
@@ -360,4 +366,61 @@ func TestEventNotCreated(t *testing.T) {
 	tc.runTest(t)
 }
 
-// TODO: add more tests
```

Comment:
Please, do not remove this message as we still need more tests, e.g., enforcement of upscal/downscale window.